src/aics-image-viewer/components/AxisClipSliders/styles.css

```diff
@@ -7,8 +7,8 @@
   --color-z: #0099ff;
 
   @extend %axis-override;
-  /* .slider-group-title (50) + .axis-slider (480) + .slider-name (10) + .slider-slices (105) + margins & padding (80) */
-  max-width: 725px;
+  /* .slider-group-title (50) + .axis-slider (480) + .slider-name (10) + .slider-values (160) + margins & padding (80) */
+  max-width: 780px;
```

Comment:
These could be true expressions if all the other numbers were stored in vars, right?  Not that we should right now but something to consider for ease of maintenance.